python/pipeline/preprocess.py

```diff
@@ -130,6 +130,21 @@ def estimate_num_components_per_slice(self):
 
             return int(round(num_components))
 
+        def estimate_neuron_size_in_pixels(self):
+            """ Estimates the size of a neuron in the scan (in pixels). Assumes neurons
+             are 15 x 15 microns."""
+            neuron_size_in_microns = 15 # assumption
+
+            # Calculate size in pixels
+            um_width, px_width = (Prepare.Galvo() & self).fetch1['um_width', 'px_width']
+            one_micron_in_pixels = px_width / um_width
+            neuron_size_in_pixels = neuron_size_in_microns * one_micron_in_pixels
+
+            # Make sure it is int and at least 1.
+            neuron_size_in_pixels = int(max(round(neuron_size_in_pixels), 1))
```

Comment:
Why round and clamp it here?  The doc string does not mention that these transformations.  I suggest returning the actual size in pixels and rounding and clamping in the calling context.